ngraph/core/reference/include/ngraph/runtime/reference/interpolate.hpp

```diff
@@ -25,6 +25,7 @@
 #include <map>
 #include "ngraph/coordinate_transform.hpp"
 #include "ngraph/shape_util.hpp"
+#include "ngraph/op/interpolate.hpp"
```

Comment:
Why do we need this in hpp?

Reply:
Because `ngraph::op::v4::Interpolate` is used in hpp